crates/re_sdk/src/spawn.rs

```diff
@@ -184,8 +184,15 @@ pub fn spawn(opts: &SpawnOptions) -> Result<(), SpawnError> {
                     executable_path: executable_path.clone(),
                 }
             } else {
+                let sdk_version = re_build_info::build_info!().version;
                 SpawnError::ExecutableNotFoundInPath {
-                    message: MSG_INSTALL_HOW_TO.to_owned(),
+                    // Only recommend a specific Viewer version for non-alpha/rc/dev SDKs.
+                    message: if sdk_version.meta.is_none() {
```

Comment:
Nit: could be clearer to expose this as a helper like `sdk_version.is_release()`.